### PR TITLE
chore(svgviewer): improve performance for svgviewer with images

### PR DESCRIPTION
**Pull request by valgussev**
For quite big SVG files with embedded images (2MB+) the performance
of SVGViewer is relatively bad.

Because willChange optimization has a calculation cost
we don't want to apply it for a bigger embedded into svg <image />
tags since it slows down the performance instead of providing
a boost. Images are already heavily optimized by browsers
so we don't need to care much about additional optimizations.

before
before

after
after

### CODE REVIEW

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-638.gearboxjs.preview.cogniteapp.com

File: src/components/SVGViewer/SVGViewer.tsx [hunk 1]
```diff
@@ -336,6 +338,16 @@ export class SVGViewer extends Component<SvgViewerProps, SvgViewerState> {
     });
   };
 
+  shouldDisableOptimization = () => {
+    // Bacause willChange optimization has a calculation cost
+    // we don't want to apply it for a bigger embedded into svg <image />
+    // tags since it slows down the performance instead of providing
+    // a boost. Images are already heavily optimizaide by browsers
```
Comment (stonecrafter) [inline src/components/SVGViewer/SVGViewer.tsx]:
```suggestion
    // a boost. Images are already heavily optimized by browsers
```

**Review by stonecrafter — APPROVED** [review]

File: src/components/SVGViewer/SVGViewer.tsx (hunk 1 above)
Comment (valgussev) [reply src/components/SVGViewer/SVGViewer.tsx]: That was one more typo :P

Comment (stonecrafter) [reply src/components/SVGViewer/SVGViewer.tsx]: I even now noticed a grammar error in those lines of code but I'll let it be 😝

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-638.gearboxjs.preview.cogniteapp.com

**Review by mrtraser — APPROVED** [review]
lgtm

**Comment by polomani** [comment]
@valgussev `chore:` will not trigger new build, you might wan't to use `perf:` instead

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-638.gearboxjs.preview.cogniteapp.com

**Comment by valgussev** [comment]
> @valgussev chore: will not trigger new build, you might wan't to use perf: instead
@polomani thanks for the heads up and fixing this

**Comment by polomani** [comment]
@valgussev thanks for pr!

**Pull request merged**